ansible-scylla-node/tasks/common.yml

```diff
@@ -277,3 +296,20 @@
   become: true
   when: update_sysctl|bool is defined and update_sysctl|bool
 
+- name: Deactivate firewall
+  block:
+    - name: Deactivate front-end firewall services
+      service:
+        name: "{{ item }}"
+        state: stopped
+        enabled: false
+      loop:
+        - firewalld
+        - iptables_services
+        - ufw
+    - name: Flush all iptables rules
+      iptables:
+        flush: yes    
+  when: not firewall_enabled
+  ignore_errors: yes
+  become: true
```

Comment:
the duplication of task is on purpose or forgotten from previous commit?

Reply:
@tarzanek Ignore this commit and use the last one. Sorry for the hassle.